### PR TITLE
#63 - Solutions block styles

### LINKED ISSUE #63: Solutions Block

### PR DESCRIPTION
**Pull request by markovukiceviccn**
Fix #63

Test URLs:
- Before: https://develop--vg-macktrucks-com-rd--netcentric.hlx.page/
- After: https://63-solutions-block--vg-macktrucks-com-rd--netcentric.hlx.page/drafts/marko/pdp

Block background colour will be delivered in this ticket: 
https://github.com/Netcentric/vg-macktrucks-com-rd/issues/64

### CODE REVIEW

File: blocks/v2-solutions/v2-solutions.css [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  @media (min-width: 1200px) {
+    font-size: var(--headline-1-font-size);
+  }
+}
+
+.v2-solutions h1 + p {
+  font: normal var(--body-font-size-s)/1.5 var(--ff-body);
+}
+
+.v2-solutions > div > div > p:first-child {
```
[the remark this answers is not in the document]
Comment (markovukiceviccn) [reply blocks/v2-solutions/v2-solutions.css]: This is now changed and use `adjustPretitle` function.

File: blocks/v2-solutions/v2-solutions.css [hunk 2]
```diff
@@ -0,0 +1,59 @@
+.v2-solutions-container {
+  display: flex;
+  flex-direction: column;
+
+  @media (min-width: 744px) {
```
[the remark this answers is not in the document]
Comment (markovukiceviccn) [reply blocks/v2-solutions/v2-solutions.css]: Media queries nesting adjusted.

File: blocks/v2-solutions/v2-solutions.css [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+.v2-solutions h1 + p {
+  font: normal var(--body-font-size-s)/1.5 var(--ff-body);
+}
+
+.v2-solutions > div > div > p:first-child {
+  font: normal var(--accent-2-font-size)/1.2 var(--ff-accents);
+}
+
+.v2-solutions .button.button--primary {
+  font-family: var(--ff-subheadings-medium);
```
[the remark this answers is not in the document]
Comment (markovukiceviccn) [reply blocks/v2-solutions/v2-solutions.css]: Font family is now changed.

File: blocks/v2-solutions/v2-solutions.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (markovukiceviccn) [reply blocks/v2-solutions/v2-solutions.css]: Done

File: blocks/v2-solutions/v2-solutions.css (hunk 3 above)
[the remark this answers is not in the document]
Comment (markovukiceviccn) [reply blocks/v2-solutions/v2-solutions.css]: This is now done

File: blocks/v2-solutions/v2-solutions.css [hunk 4]
```diff
@@ -0,0 +1,65 @@
+.v2-solutions-container {
+  display: flex;
+  flex-direction: column;
+}
+
+@media (min-width: 744px) {
```
[the remark this answers is not in the document]
Comment (markovukiceviccn) [reply blocks/v2-solutions/v2-solutions.css]: It is now grouped.

File: blocks/v2-solutions/v2-solutions.css [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+  .v2-solutions h1 {
+    font-size: var(--headline-1-font-size);
+  }
+}
+
+.v2-solutions h1 + p {
+  font: normal var(--body-font-size-s)/1.5 var(--ff-body);
+}
+
+.v2-solutions .pretitle {
```
[the remark this answers is not in the document]
Comment (markovukiceviccn) [reply blocks/v2-solutions/v2-solutions.css]: This is now done.

**Review by TomaszDziezykNetcentric — APPROVED** [review]

**Pull request merged**